docs/API.md

```diff
@@ -1376,6 +1399,10 @@ newer revisions. The fully qualified ids of those charms will be returned in an
 ordered list from newest to oldest revision. Note that the current revision
 will be included in the list as it is also an available revision.
 
+If a charm supports multiple series, and the series is not specified in the
```

Comment:
This would be awkward to implement as the endpoints don't actually know whether the series was specified in the request or not.

I think that if a charm supports multiple series, the resolved URL will omit the series, so the results would be more like:

	Example: `GET wordpress/meta/revision-info`
	
	```json
	{
	    "Revisions": [
	        "cs:wordpress-3",
	        "cs:wordpress-2",
	        "cs:wordpress-1",
	    ]
	}

It's an open question whether the endpoint should provide earlier revisions of legacy series-specific entities. Perhaps the answer should be yes - it could provide all earlier revisions with series compatible with the resolved URL.

Given that we don't actually use this endpoint, the point is somewhat moot. Perhaps just leave a TODO?